src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -147,6 +147,21 @@ export const removeConceptMapping = (data, source) => dispatch => {
     });
 };
 
+export const backgroundRemoveConceptMapping = (data) => dispatch => (
+   api.dictionaries
+    .removeConceptMapping(data)
+    .then(
+      () => {
+        dispatch(removeMapping(data.references[0]));
+        const ConceptsToFetch = `/users/${localStorage.getItem('username')}/sources/${localStorage.getItem('dictionaryId')}/concepts/?includeMappings=true&q=&limit=0&page=1&verbose=true`
+          dispatch(fetchDictionaryConcepts(ConceptsToFetch));
+      }
+    )
+    .catch((error) => {
```

Comment:
Will this be always a network error?

Reply:
The network error should only be triggered if there's no response from the api.